### ISSUE TITLE
Show URL in bottom left corner on link hover

### ISSUE THREAD

**Issue by Larivact** [typora/typora-issues#572]
This is the way all major web browsers do it and not having it is irritating.

**Comment by AlexPasternak** [comment]
Oh yes, I was just thinking about this yesterday. You can't tell what the URL is without clicking into it, which is a pain.

**Comment by arieljannai** [comment]
Just wanted to open an issue about it 😄

**Comment by abnerlee** [comment]
relates https://github.com/typora/typora-issues/issues/277

**Comment by vr8hub** [comment]
Yes! (Although I don't care where it displays; where the cursor is like footnotes would be fine, too.)
